SPECS/glusterfs/glusterfs.spec

```diff
@@ -74,74 +81,116 @@
 ##-----------------------------------------------------------------------------
 ## All %%global definitions should be placed here and keep them sorted
 ##
+# selinux booleans whose defalut value needs modification
+# these booleans will be consumed by "%%selinux_set_booleans" macro.
+%global _with_systemd true
 %global _with_firewalld --enable-firewalld
-%if ( 0%{?_tmpfilesdir:1} )
+%if 0%{?_tmpfilesdir:1}
 %global _with_tmpfilesdir --with-tmpfilesdir=%{_tmpfilesdir}
 %else
 %global _with_tmpfilesdir --without-tmpfilesdir
 %endif
 # without server should also disable some server-only components
-%if ( 0%{?_without_server:1} )
+%if 0%{?_without_server:1}
 %global _without_events --disable-events
 %global _without_georeplication --disable-georeplication
+%global _without_linux_io_uring --disable-linux-io_uring
 %global _with_gnfs %{nil}
 %global _without_ocf --without-ocf
 %endif
+%global _usepython3 1
```

Comment:
We don't really have other options in our case. In such cases we can just remove the macro and wherever it's used assume it was set to `1` to clean-up any `%if` macros as well.